libraries/AP_NavEKF3/AP_NavEKF3_AirDataFusion.cpp

```diff
@@ -475,8 +475,8 @@ void NavEKF3_core::FuseSideslip()
 void NavEKF3_core::FuseDragForces()
 {
     // drag model parameters
-    const ftype bcoef_x = frontend->_ballisticCoef_x;
-    const ftype bcoef_y = frontend->_ballisticCoef_y;
+    const ftype bcoef_x = frontend->_ballisticCoef_x.get();
+    const ftype bcoef_y = frontend->_ballisticCoef_y.get();
```

Comment:
I wonder if these would be clearer as casts-to-ftype rather than .get()

similarly for many of the other casts.

Reply:
I think ".get()" is prettier than a cast..

Reply:
It would also have to be two casts, first to `float` then to the desired type, as we probably don't wish to implement cast operators for all desired types.